src/helpers/file.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
What that means?

Reply:
It's generated from the compilation to Javascript, it allow us to define some restrict definitions, you can check the explanation [here](https://pt.stackoverflow.com/a/3752/71065).